packages/react/package.json

```diff
@@ -85,6 +85,9 @@
     "extends": [
       "@uploadthing/eslint-config/base",
       "@uploadthing/eslint-config/react"
-    ]
+    ],
+    "rules": {
+      "no-console": "off"
```

Comment:
added these as i am unsure if we should ship these kb's to the browser or keep it server-only?

Reply:
probably server only?